phpstan.neon

```diff
@@ -6,6 +6,7 @@ includes:
 	- vendor/phpstan/phpstan/conf/config.levelmax.neon
```

Comment:
We are using the maximum level already, @szepeviktor!

Reply:
Oh! Shoot! :)

that line was hiding there.

Reply:
Ha, no problem!

Thank you for taking a look, @szepeviktor!